api/README.md

```diff
@@ -753,3 +754,80 @@ empty string, which will yield all test results for the selected runs.
 ```
 
 </details>
+
+## Metadata results
+
+### /api/metadata
+
+This endpoint accepts POST and GET requests.
+
+- GET request returns Metadata Link Information by product, and requires product parameters;
+
+- POST request searches Metadata Link by link url, and requires product parameters and payload.
+
+__URL Parameters__
+
+__`product`__ : browser[version[os[version]]]. e.g. `chrome-63.0-linux`
+
+#### JSON Request Payload
+```json
+[
+  {
+    "link": [pattern]
+  }
+]
+```
+
+  Where `[pattern]` is a susbtring of the url field of a Metadata Link Node.
+
+#### Get Examples
+
+- /api/metadata?product=chrome&product=safari
+
+<details><summary><b>Example JSON</b></summary>
+
+```json
+[
+   {
+      "test":"/IndexedDB/bindings-inject-key.html",
+      "urls":[
+         "bugs.chromium.org/p/chromium/issues/detail?id=934844",
+         ""
```

Comment:
what's with the empty strings? (here and L801)

Reply:
I added a url link for safari. It was empty because this test didn't have a corresponding bug link in safari. But I guess if it is just for demonstration, the link doesn't have to be real?